src/styles/main.scss

```diff
@@ -1,3 +1,11 @@
+@import "../../src/styles/variables.scss";
+@import "../../src/styles/modifiers.scss";
+@import "../../src/styles/calendar.scss";
+
+// * {
+//   box-sizing: border-box;
+// }
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)